projects_using_gazebo.md

```diff
@@ -4,27 +4,31 @@ This is an incomplete list of projects that use (modern) Gazebo. It includes
 plugins, examples, ROS integrations, etc. If you'd like to add your project,
 please open a pull request by clicking on the "Edit on Github" link.
 
-## Gazebo Plugins
+## Gazebo Tools for Integration
+
+- **ros_gz_project_template**: A template project integrating ROS and Gazebo
+  simulator. https://github.com/gazebosim/ros_gz_project_template
 
 ## Gazebo and ROS 2 Integration
 
+- **Aerostack2**: An open-source autonomous multi-aerial-robots systems framework. [{octicon}`globe` link](https://github.com/aerostack2/aerostack2)
+- **ArduPilot**: ArduPilot is an autopilot built on ChibiOS RTOS that supports many vehicle types. [{octicon}`globe` link](https://ardupilot.org/dev/docs/sitl-with-gazebo.html)
+- **CogniPilot**: CogniPilot is an autopilot built on Zephyr RTOS that supports many vehicle types. [{octicon}`globe` link](https://cognipilot.org)
+  | <https://github.com/CogniPilot/synapse_gz>
 - **gz_ros2_control**: ROS 2 package for integrating `ros2_control` and Gazebo.
   [{octicon}`globe` link](https://control.ros.org/jazzy/doc/gz_ros2_control/doc/index.html)
   | <https://github.com/ros-controls/gz_ros2_control>
-- **Nav2**: ROS 2 Navigation framework and system.
-  [{octicon}`globe` link](https://nav2.org/) |
-  <https://github.com/ros-navigation/navigation2>
-- **ros_gz_project_template**: A template project integrating ROS and Gazebo
-  simulator. https://github.com/gazebosim/ros_gz_project_template
-- **Aerostack2**: An open-source autonomous multi-aerial-robots systems framework. [{octicon}`globe` link](https://github.com/aerostack2/aerostack2)
+- **heightmap_spawner**: A ROS 2 package that converts occupancy grids from nav2_map_server into 3D heightmap models for use in Gazebo simulations. [{octicon}`globe` link](https://github.com/damanikjosh/heightmap_spawner)
 - **MBARI Buoy Sim**: Wave simulation for energy buoy control. [{octicon}`globe` link](https://github.com/osrf/buoy_sim)
 - **MBARI LRAUV**: Underwater simulation for glider science missions. [{octicon}`globe` link](https://github.com/osrf/lrauv)
 - **MBZIRC**: Maritime competition for surface and aerial vehicles with manipulation. [{octicon}`globe` link](https://github.com/osrf/mbzirc)
+- **Nav2**: ROS 2 Navigation framework and system.
+  [{octicon}`globe` link](https://nav2.org/) |
+  <https://github.com/ros-navigation/navigation2>
+- **Open-RMF**: Warehouse robot fleet sharing and interoperability between multiple fleets of robots and physical infrastructure. [{octicon}`globe` link](https://www.open-rmf.org/)
+- **PhantomX Pincher Robot Arm**: Manipulation packages in ROS 2, MoveIt 2, and new Gazebo for a class at the University of Luxembourg. [{octicon}`globe` link](https://github.com/snt-spacer/phantomx_pincher/tree/ros2)
+- **PX4**: PX4 is an autopilot built on Nuttx RTOS that supports many vehicle types. [{octicon}`globe` link](https://docs.px4.io/v1.13/en/simulation/ignition_gazebo.html)
```

Comment:
Actually, can we change the link to https://docs.px4.io/main/en/sim_gazebo_gz/. I'd avoid links pinned to a version.